packages/sync-service/lib/electric/schema.ex

```diff
@@ -0,0 +1,95 @@
+defmodule Electric.Schema do
+  import Bitwise
+
+  @bit_types ["bit", "varbit"]
+  @variable_length_character_types ["varchar", "text"]
+  @fixed_length_character_types ["bpchar"]
+  @time_types ["time", "timetz", "timestamp", "timestamptz", "interval", "date"]
```

Comment:
nitpick: Can you pull this info from `Electric.Replication.Eval.Env.BasicTypes` maybe?